src/adapters/prebidServer.js

```diff
@@ -21,9 +45,39 @@ function PrebidServer() {
     config = s2sconfig;
   };
 
+  function convertTypes(adUnits) {
+    adUnits.forEach(adUnit => {
+      adUnit.bids.forEach(bid => {
+        const types = paramTypes[bid.bidder] || [];
+        Object.keys(types).forEach(key => {
+          if(bid.params[key] && typeof bid.params[key] !== types[key]) {
+            //mismatch type. Try to fix
+            utils.logMessage(`Mismatched type for Prebid Server : ${bid.bidder} : ${key}. Required Type:${types[key]}`);
+            bid.params[key] = tryConvertType(types[key], bid.params[key]);
+            //don't send invalid values
+            if(!bid.params[key]) {
+              delete bid.params.key;
+            }
+          }
+        });
+      });
+    });
+  }
+
+  function tryConvertType(typeToConvert, value) {
+    if(typeToConvert === 'string') {
+      return Number.toString(value);
```

Comment:
I was going to say "weren't we requiring 80% unit test coverage on all new code?" But... dunno if that counts